main.js

```diff
@@ -39,12 +71,35 @@ function turnCar() {
 function startAndStopCar() {
 
   if (event.keyCode === 32 && data.started === false) {
-    intervalId = setInterval(function () {
-      var car = document.querySelector('img');
-      data.positionX += 10;
-      car.style.left = data.positionX + 'px';
-    }, 16);
-    data.started = true;
+    if (data.facing === 'east') {
+      intervalId = setInterval(function () {
+        var car = document.querySelector('img');
+        data.positionX += 10;
+        car.style.left = data.positionX + 'px';
+      }, 16);
```

Comment:
This sort of function gets repeated a lot in your code, you should instead make one function to call that moves the car in a direction based on the current value of `data.facing`